servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -232,12 +232,13 @@ def format_scattered_task(task_name, task_metadata):
             else:
                 temp_status_collection[
                     status] = temp_status_collection[status] + 1
-        if minStart > _parse_datetime(shard.get('start')):
-            minStart = _parse_datetime(shard.get('start'))
-        if shard.get('executionStatus') not in ['Failed', 'Done']:
-            maxEnd = None
-        if maxEnd is not None and maxEnd < _parse_datetime(shard.get('end')):
-            maxEnd = _parse_datetime(shard.get('end'))
+            if minStart > _parse_datetime(shard.get('start')):
```

Comment:
Could you remind me why move this conditional logic a level deeper?

Reply:
it was moved under the `if current_shard != shard.get('shardIndex'):` check to only get the data from the latest attempts of each shard

Reply:
Thanks for the explanation, that makes sense!